### PR TITLE
Style clangformat

### PR DESCRIPTION
**Pull request by PFigs**

### CODE REVIEW

File: lib/api/wpc.h [hunk 1]
```diff
@@ -39,49 +39,51 @@ typedef uint8_t net_channel_t;
...
 typedef enum
 {
-    APP_ROLE_SINK = 1,    //!< Node is sink
-    APP_ROLE_HEADNODE = 2,//!< Node is headnode
-    APP_ROLE_SUBNODE = 3, //!< Node is subnode (no routing of other's traffic)
-    APP_ROLE_UNKNOWN      //!< Erroneous role
+    APP_ROLE_SINK = 1,        //!< Node is sink
+    APP_ROLE_HEADNODE = 2,    //!< Node is headnode
+    APP_ROLE_SUBNODE =
+        3,              //!< Node is subnode (no routing of other's traffic)
```
Comment (GwendalRaoul) [inline lib/api/wpc.h]: Not a huge fan of this refactoring but I think only solution is to reduce a bit the comment length

File: lib/api/wpc.h [hunk 2]
```diff
@@ -101,55 +103,60 @@ typedef enum
...
+    APP_RES_STACK_IS_STOPPED,         //!< Stack is stopped
+    APP_RES_NODE_NOT_A_SINK,          //!< Node is not a sink
+    APP_RES_UNKNOWN_DEST,             //!< Unknown destination address
+    APP_RES_NO_CONFIG,                //!< No configuration received/set
+    APP_RES_ALREADY_REGISTERED,       //!< Cannot register several times
+    APP_RES_NOT_REGISTERED,       //!< Cannot unregister if not registered first
+    APP_RES_ATTRIBUTE_NOT_SET,    //!< Attribute is not set yet
+    APP_RES_ACCESS_DENIED,        //!< Access denied
+    APP_RES_DATA_ERROR,           //!< Error in data
+    APP_RES_NO_SCRATCHPAD_START,        //!< No scratchpad start request sent
```
Comment (GwendalRaoul) [inline lib/api/wpc.h]: New indentation seems weird.

File: lib/platform/linux/platform.c [hunk 3]
```diff
@@ -298,20 +302,14 @@ bool Platform_init()
     }
 
     // Start a thread to poll for indication
-    if (pthread_create(&thread_polling,
-                       NULL,
-                       poll_for_indication,
-                       NULL) < 0)
+    if (pthread_create(&thread_polling, NULL, poll_for_indication, NULL) < 0)
```
Comment (GwendalRaoul) [inline lib/platform/linux/platform.c]: I prefer the old version that is more readable

File: lib/wpc/attribute.c [hunk 4]
```diff
@@ -24,20 +24,26 @@ int attribute_write_request(uint8_t primitive_id,
         return -1;
 
     request.primitive_id = primitive_id;
-    uint16_encode_le(attribute_id,
-        (uint8_t *)&(request.payload.attribute_write_request_payload.attribute_id));
-    request.payload.attribute_write_request_payload.attribute_length = attribute_length;
-    memcpy(request.payload.attribute_write_request_payload.attribute_value, attribute_value_p, attribute_length);
+    uint16_encode_le(
```
Comment (GwendalRaoul) [inline lib/wpc/attribute.c]: Not really readable

File: lib/wpc/include/dsap.h [hunk 5]
```diff
@@ -107,9 +107,8 @@ int dsap_data_tx_request(const uint8_t * buffer,
                          uint8_t hop_limit);
 
 /**
- * \brief   Handler for tx indication. It is called when sent data leaves the node
- * \param   payload
- *          Pointer to payload
+ * \brief   Handler for tx indication. It is called when sent data leaves the
```
Comment (GwendalRaoul) [inline lib/wpc/include/dsap.h]: Strange change

File: lib/wpc/slip.c [hunk 6]
```diff
@@ -125,35 +118,35 @@ static uint32_t slip_decode_buffer(uint8_t* buffer,
...
     }
     return write;
 }
 
-int Slip_decode(uint8_t * buffer,
-                uint32_t len)
+int Slip_decode(uint8_t * buffer, uint32_t len)
 {
     int decoded_len;
     uint16_t crc, crc_from_frame;
```
Comment (teemupiiroinenwirepas) [inline lib/wpc/slip.c]: Should notice two variable definitions and one type. Should be:

uint16_t crc;
uint16_t crc_from_frame;

File: example/main.c [hunk 7]
```diff
@@ -46,43 +46,42 @@ static bool onDataReceived(const uint8_t * bytes,
...
     }
 
-    if (argc > 2)
+    if(argc > 2)
     {
         bitrate = strtoul(argv[2], NULL, 0);
     }
 
-    if (WPC_initialize(port_name, bitrate) != APP_RES_OK)
+    if(WPC_initialize(port_name, bitrate) != APP_RES_OK)
```
[the remark this answers is not in the document]
Comment (PFigs) [reply example/main.c]: seems it fails to detect it, even though the configuration seems to disallow it

File: lib/api/logger.h [hunk 8]
```diff
@@ -55,27 +56,28 @@ extern void Platform_LOG(char level, char * module, char * format, va_list args)
...
+#define ERROR_LOG_LEVEL 0
+#define NO_LOG_LEVEL -1
 
 /**
  * Only logs with a level lower or equal to MAX_LOG_LEVEL will be printed
  *
  * \note this constant can be defined in each file including this file
  */
 #ifndef MAX_LOG_LEVEL
-    /* By default only errors are displayed */
```
[the remark this answers is not in the document]
Comment (PFigs) [reply lib/api/logger.h]: I agree, I though the PD was not available in version 7 (stable) but it seems we can at least have

#ifdef 
#    ...
#endif 

Not ideal, but better than not having any indent

File: lib/wpc/attribute.c [hunk 9]
```diff
@@ -20,24 +20,27 @@ int attribute_write_request(uint8_t primitive_id,
...
     wpc_frame_t request, confirm;
 
-    if (attribute_length > 16)
+    if(attribute_length > 16)
         return -1;
 
     request.primitive_id = primitive_id;
     uint16_encode_le(attribute_id,
-        (uint8_t *)&(request.payload.attribute_write_request_payload.attribute_id));
+                     (uint8_t *) &(
```
[the remark this answers is not in the document]
Comment (PFigs) [reply lib/wpc/attribute.c]: @GwendalRaoul said exactly the same. This one must receive a manual override.

File: lib/wpc/attribute.c [hunk 10]
```diff
@@ -20,24 +20,27 @@ int attribute_write_request(uint8_t primitive_id,
...
+    res = WPC_Int_send_request(&request, &confirm);
 
-    if (res < 0)
-       return res;
+    if(res < 0)
+        return res;
 
-    LOGD("Attribute write result = %d\n", confirm.payload.sap_generic_confirm_payload.result);
+    LOGD("Attribute write result = %d\n",
+         confirm.payload.sap_generic_confirm_payload.result);
```
[the remark this answers is not in the document]
Comment (PFigs) [reply lib/wpc/attribute.c]: does not seem to be available as a configuration target

**Pull request merged**